### ISSUE TITLE
Features I'd love to see added

### ISSUE THREAD

**Issue by Clasurol** [dremin/RetroBar#395]
Retrobar is an outstanding taskbar project. You've knocked it out of the park, you really have. I know it's still an evolving piece of software, so here's a list of features I'd love to see implementing sometime down the road (please note this is not a request, merely stuff I think would be awesome to see/simply a wish list):

- Add rebars to the quick launch area that can be dragged to shorten or extend said area. Also add the ability to "lock" the taskbar.
-  Allow the systray icons to be dragged and dropped. Also would be great if they were remembered in the order they were left in.
- Add a calendar, which IMO would be ideal if you invoked by left clicking on the clock area. In terms of design, I was thinking something akin to this:
![win95-1-1](https://user-images.githubusercontent.com/46195405/181994903-61d83a2e-ac4d-4d5e-be04-6356fec69f05.png)
- "EverythingToolbar" toolbar support (see this Github page for more info: https://github.com/stnkl/EverythingToolbar).
- Add the ability to hide programs from showing in the taskbar, but still allow them to show elsewhere (in alt-tab menu, and their window is still displayable). Basically the same as you can do for the system tray/notification area.
- Media player toolbar, to allow you to basic playback functionality (play/pause, rewind/forward, etc).
- Add the ability to allow the taskbar to autohide itself like the default taskbar can.

Again, this is a wish list, nothing more. Thank you for taking the time to read this, and have a good day.

Many thanks, 

Rob.

**Comment by ThaBluNate** [comment]
## +1 
Locking the taskbar and setting auto-hide time
Organized systray would be a blessing

## -1
I still don't know why you requested a calendar? Kind of weird considering this is a taskbar program.
Media player toolbar is most likely impossible (sorry)

## ...1
I wouldn't know enough about EverythingToolbar to say anything, leaving the post at exactly 0 points from me

**Comment by Clasurol** [comment]
> I still don't know why you requested a calendar? Kind of weird considering this is a taskbar program.

Not really. A classic-styled calendar applet would go well with the taskbar as a nice feature IMO.

**Comment by SL6464** [comment]
A classic calendar would be amazing, especially given that clicking the clock in RetroBar currently just brings up the Action Centre.
